shared/src/main/kotlin/com/egm/stellio/shared/util/NgsiLdParsingUtils.kt

```diff
@@ -246,6 +258,15 @@ object NgsiLdParsingUtils {
             values[NGSILD_ENTITY_TYPE] is List<*> &&
             values.getOrElse(NGSILD_ENTITY_TYPE) { emptyList() }[0] == type.uri
 
+    /**
+     * Given an entity's attribute, returns whether all its instances are of the given attribute type
+     * (i.e. property)
+     */
+    fun isAttributeOfType(values: List<Map<String, List<Any>>>, type: AttributeType): Boolean =
```

Comment:
This is a little bit weird. What if a multi-attribute is a mix of properties and relationships? (yes, it should be rejected)

Reply:
indeed

Reply:
Here we are checking if all attribute instances are of type property.
If not we return false and the attribute will be rejected.

Reply:
rejected or silently ignored?

Reply:
By rejected you mean an exception should be raised and the entity cannot be created ?